### ISSUE TITLE
Utilization of existing standards

### ISSUE THREAD

**Issue by jybird47** [GSA/modernization#7]
In the existing report is the recommendation to encrypt data at rest.  ANSI and ISO and other standards generating organizations have published peer reviewed documentation on this subject.  Specifically, ANSI X9.69 Framework for Key Management and X9.73 Cryptographic Message Syntax as well as ISO 11568 CMS, all address the issue of persistent protection of data.  The creation of self protecting data objects rather than depending on network topography not controlled by the information owner.  NIST is a member of ANSI.  And ANSI is the American vote to ISO.

**Comment by johnaweiler** [comment]
Agreed, an area where Federal IT is greatly lacking in spite of  OMB A119 directive to support, adopt and fund SDOs.  The IT-AAC has made this much simplier by forming a public/private partnership with over two dozen SDOs and NGOs reaching over 100,000 innovative organizations who are driving the $4T Global IT market, of which federal IT is less than 2 percent and worst performing sector.
